pkg/hub/submarineragent/controller.go

```diff
@@ -77,9 +78,9 @@ var sccFiles = []string{
 	"manifests/rbac/submariner-agent-scc.yaml",
 }
 
-var operatorFiles = []string{
-	"manifests/operator/submariner-operator-group.yaml",
-	"manifests/operator/submariner-operator-subscription.yaml",
+var operatorFiles = map[string]string{
+	operatorGroup:       "manifests/operator/submariner-operator-group.yaml",
+	"subscriptionGroup": "manifests/operator/submariner-operator-subscription.yaml",
```

Comment:
I’m not convinced by this approach — it requires both a specific data structure _and_ supporting code.

If you want to keep the idea of lists of files, how about

```
var operatorAllFiles = [] string {
    "manifests/operator/submariner-operator-group.yaml",
    "manifests/operator/submariner-operator-subscription.yaml",
}

var operatorSkipFiles = []string {
    "manifests/operator/submariner-operator-subscription.yaml",
}

...

if skipOperatorGroup {
    files = append(files, operatorSkipFiles)
} else {
    files = append(files, operatorAllFiles)
}
```

Or we could make the whole thing fully generic: make a “skip” annotation which is a list of files to skip, and when populating the lists of files, check that each file isn’t in the annotation.

Reply:
I like the two vars with different set of files idea. I want to add skip annotation with files coz it adds more work for devs and needs them to be aware of what are the file paths to use. We can go that way if we tomorrow have more files and want to let devs pick and choose.